build.gradle

```diff
@@ -40,10 +43,17 @@ allprojects {
         mavenCentral()
         maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
     }
+
+    tasks.withType(JavaCompile) {
+        options.encoding = 'UTF-8'
+    }
+
+    tasks.withType(Test) {
+        systemProperty 'file.encoding', 'UTF-8'
+    }
 }
 
 configure(subprojects - project(":tests:gdx-tests-android")) {
-    apply plugin: "idea"
```

Comment:
Why is the `idea` plugin getting removed here? Is it no longer needed?

Reply:
It got moved to where I added eclipse, so that all projects get it.

https://github.com/libgdx/libgdx/blob/3defef117ac6ffd328788d634fc2162c9f300525/build.gradle#L20